homeassistant/components/modbus/modbus.py

```diff
@@ -316,7 +316,7 @@ async def async_pb_connect(self) -> None:
                 self._log_error(err, error_state=False)
                 return
             message = f"modbus {self.name} communication open"
-            _LOGGER.warning(message)
+            _LOGGER.info(message)
```

Comment:
I think they should be `debug` messages - not `info`.

Reply:
> I think they should be `debug` messages - not `info`.

Actually, cancel that!
Looks like `info` is good after all: https://github.com/home-assistant/core/pull/125939#issuecomment-2401515479